mbuild/tests/test_compound.py

```diff
@@ -15,6 +15,33 @@ class TestCompound(BaseTest):
     def test_load_and_create(self):
         mb.load(get_fn('methyl.pdb'))
 
+    def test_load_conversion(self,ethane,h2o):
+        compound = mb.Compound([ethane,h2o])
+        parm = compound.to_parmed()
+        traj = compound.to_trajectory()
+
+        for topo in [compound,parm,traj]:
+            topo_converted = mb.load(topo)
+            assert isinstance(topo_converted, mb.Compound)
+            assert topo_converted.n_particles == 11
+            assert len([at for at in topo_converted.particles() if at.name == 'C']) == 2
+            assert len([at for at in topo_converted.particles() if at.name == 'H']) == 8
+            assert len([at for at in topo_converted.particles() if at.name == 'O']) == 1
+
```

Comment:
This is probably overkill but could we have a test that assures the outputs of `mycompound = mb.load(mystruct)` and `mycompound = mb.Compound(); mycompound.from_parmed(mystruct)` are identical (maybe with some kwargs)?

Reply:
I can add that test, but it will not be a direct comparison like `mycompound_from_mb == mycompound_from_pmd`, since those two structures will never be identical (they will have different ID). However, I think I can extract information from those structures and make sure they match.